MidasCivil_Adapter/PrivateHelpers/ReadResultAPI.cs

```diff
@@ -0,0 +1,201 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2025, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.Net.Http;
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net;
+using BH.oM.Analytical.Results;
+using BH.oM.Structure.Results;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Linq;
+using BH.oM.Data.Requests;
+
+
+namespace BH.Adapter.MidasCivil
+{
+    public partial class MidasCivilAdapter
+    {
+        private async Task<IEnumerable<IResult>> ExtractResultAPI(string resultType, List<int> ids, List<string> loadcaseIds)
+        {
+            List<IResult> results = new List<IResult>();
+
+            const string endpoint = "post/TABLE";
+
+            string jsonPayload = "";
+
+            string units = "\"UNIT\": {\"FORCE\": \"N\", \"DIST\": \"m\"}, ";
+            string format = "\"STYLES\": {\"FORMAT\": \"Fixed\", \"PLACE\": 3}";
```

Comment:
```suggestion
            string format = "\"STYLES\": {\"FORMAT\": \"Fixed\", \"PLACE\": 3}";
```
Are these SF or DP? Tempted to do 6DP and then users can decide whether to change.